### PR TITLE
OCM-10299 | ci: fix rosa-hcp-external-auth deploy failure

### PR DESCRIPTION
**Pull request by yingzhanredhat**
https://privatebin.corp.redhat.com/?6928f4a1036740de#125p1Gdi4S4EGXx2bryyaUKVQE3PYpHktNQ5JxvkrFfV

### CODE REVIEW

File: tests/e2e/e2e_setup_test.go [hunk 1]
```diff
@@ -46,13 +47,19 @@ var _ = Describe("Cluster preparation", labels.Feature.Cluster, func() {
```
Comment (xueli181114) [inline tests/e2e/e2e_setup_test.go:47]: use `path.Join` instead of fmt.Sprintf

**Comment by aaraj7** [comment]
Test cases are failing, these two jobs must not be made required

**Comment by xueli181114** [comment]
/override pull-ci-openshift-rosa-master-e2e-presubmits-pr-rosa-sts-advanced

**Comment by xueli181114** [comment]
/override pull-ci-openshift-rosa-master-e2e-presubmits-pr-rosa-sts-advanced

**Pull request merged**